caffe2/core/nomnigraph/Converters/Caffe2.cc

```diff
@@ -0,0 +1,327 @@
+#include "nomnigraph/Converters/Caffe2.h"
+#include "nomnigraph/Graph/Algorithms.h"
+
+#include "nomnigraph/Support/Casting.h"
+#include "nomnigraph/Support/Pointer.h"
+
+std::map<std::string, caffe2::Argument>
+getArgumentsFromOperator(caffe2::OperatorDef op) {
+  std::map<std::string, caffe2::Argument> argMap;
```

Comment:
I'm not sure building a map `arg name -> arg object` doesn't loose information: probably you could have multiple arguments with the same name in place of an argument with repeated value. I haven't seen such models, but have seen code that handles such situations. cc @Yangqing

Reply:
If doing an `arg name -> arg object` map  is fine, this function probably should be in `utils/proto_utils.h`

Reply:
this is how Glow does it I believe